lib/myads/mybanner.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:google_mobile_ads/google_mobile_ads.dart';
+
+class MyBannerAds {
```

Comment:
name the file with the name of class but in snake case. 
for this case: my_banner_ads